src/coredump/coredump.c

```diff
@@ -63,6 +64,10 @@
 #define JOURNAL_SIZE_MAX ((size_t) (10LU*1024LU*1024LU))
 #endif
 
+#define DEFAULT_COREDUMP_RATELIMIT_INTERVAL 0
+
+#define DEFAULT_COREDUMP_RATELIMIT_BURST    0
```

Comment:
pretty useless `#define`s, you just use them once, and they might as well be configured there

Reply:
Addressed the comment. Please check once.